### ISSUE TITLE
Jersey Logging filter cuts entity - placeholder for #3306 and #3281

### ISSUE THREAD

**Issue by jansupol** [eclipse-ee4j/jersey#4034]
#3281 (Jersey Logging filter cuts entity)
#3306 (Please reopen the task)

**Comment by dkBrazz** [comment]
I made a fix - #4753
Is there any chance that it will be merged into master?

**Comment by senivam** [comment]
@dkBrazz yes, sure, we will review it soon. Thanks for the PR, it's appreciated.
